packages/web_benchmarks/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 2.0.0
 
 * Updates minimum supported SDK version to Flutter 3.16/Dart 3.2.
+* Adds support for running benchmarks with the wasm compilation target.
 
```

Comment:
This needs a description of the breaking changes; see https://github.com/flutter/flutter/blob/master/docs/ecosystem/contributing/README.md#changelog-style